daemon/integration/utils.go

```diff
@@ -0,0 +1,76 @@
+// Utilities for integration testing Scoot Daemon
+package integration
+
+import (
+	"bytes"
+	"github.com/scootdev/scoot/daemon/client/cli"
+	"io"
+	"os"
+)
+
+// Run cl with args, return its output
+func Run(cl *cli.CliClient, args ...string) (string, string, error) {
+	oldArgs := os.Args
+	os.Args = append([]string{"scootcl"}, args...)
+	defer func() {
+		os.Args = oldArgs
+	}()
+
+	output := captureOutput()
+	defer output.Reset()
+
+	err := cl.Exec()
+	stdout, stderr := output.WaitAndReset() // Reset early so we can use stdout/stderr and write to uncaptured stdout/stderr
+	return stdout, stderr, err
+}
+
+// Below here is code to capture stdout and stderr
+type output struct {
+	stdout    *os.File
+	stdoutCh  chan string
+	stderr    *os.File
+	stderrCh  chan string
+	oldStdout *os.File
+	oldStderr *os.File
+}
+
+func captureOutput() *output {
+	oldStdout, oldStderr := os.Stdout, os.Stderr
+	stdout, stdoutCh := makeDiversion()
+	stderr, stderrCh := makeDiversion()
+	os.Stdout, os.Stderr = stdout, stderr
+	return &output{stdout, stdoutCh, stderr, stderrCh, oldStdout, oldStderr}
+}
+
+func makeDiversion() (*os.File, chan string) {
```

Comment:
Added more comments to this file; WDYT?

(Note: this is not new code; this is moved over from cli_test.go to be shared by both client and integration tests)